backend/index.js

```diff
@@ -1,10 +1,86 @@
-const { PrismaClient } = require('@prisma/client')
-const prisma = new PrismaClient()
-const cors = require('cors')
-const express = require('express')
+// const { PrismaClient } = require('@prisma/client')
```

Comment:
Let's remember to not include commented code in code commits -- totally fine to have them locally, but just remember to remove them when pushing a commit to GitHub